app/src/main/java/com/duckduckgo/app/global/model/Site.kt

```diff
@@ -26,24 +28,38 @@ import com.duckduckgo.app.trackerdetection.model.TrackingEvent
 
 interface Site {
 
-    val url: String
-    val privacyPractices: PrivacyPractices.Practices
-    val memberNetwork: TrackerNetwork?
+    /*
+     * The current url for this site. This is sometimes different to the url originally
+     * loaded as the url may change while loading a site
+     */
+    var url: String
```

Comment:
do we need both `Site.uri` and `Site.url`? Can these differ from each other, or will they always be the same underlying address?

Reply:
We use both so I have included both for convenience as it seemed messy to build the Uri each time it was needed (and a bit inefficient). They will always be in sync as the url setter updates the uri and the interface for uri doesn't allow setting. That said, I have added a private backing field to make it safer again.